doc/contributor/howto-guide-adding-generated-libraries.md

```diff
@@ -216,7 +219,6 @@ running while you work on tweaks to the quickstart and documentation.
 
 ```shell
 env GOOGLE_CLOUD_CPP_CHECK_API=${library} ci/cloudbuild/build.sh -t check-api-pr
-git add ci/abi-dumps
```

Comment:
I think for new libraries the commit will do nothing without a `git add`.

Reply:
Yeah, you're right. Fixed. Thanks.